docs/gatsby-config.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
           ],
           Appendices: [
             'proxy-configuration',
-            'migration-two-dot',
+            'installing-graphql-tools',
```

Comment:
Unrelated to the `graphql-tools` article, feels safe to sunset the "Migrating from 1.0" article (URL will still work)